app/code/Magento/Widget/view/adminhtml/templates/instance/edit/layout.phtml

```diff
@@ -487,7 +487,7 @@ window.WidgetInstance = WidgetInstance;
 
 jQuery(function(){
     <?php foreach ($block->getPageGroups() as $pageGroup): ?>
-        WidgetInstance.addPageGroup(<?php /* @noEscape */ echo Zend_Json::encode($pageGroup) ?>);
+        WidgetInstance.addPageGroup(<?php /* @noEscape */ echo $this->helper('Magento\Framework\Json\Helper\Data')->jsonEncode($pageGroup) ?>);
```

Comment:
This class is deprecated.
Please change to `\Magento\Framework\Serialize\Serializer\Json`